### PR TITLE
Fix issue 200

### PR DESCRIPTION
**Pull request by roigcarlo**
This PR implements #200

### CODE REVIEW

File: simphony/cuds/abc_modeling_engine.py [hunk 1]
```diff
@@ -86,6 +86,12 @@ def get_dataset(self, name):
         """
 
     @abstractmethod
+    def get_dataset_names(self):  # pragma: no cover
+        """ Returns the names of all the CUDS objects in the containers.
```
Comment (itziakos) [inline simphony/cuds/abc_modeling_engine.py]: please change to " Returns the names of the all the datasets in the engine workspace. "

File: simphony/io/h5_cuds.py [hunk 2]
```diff
@@ -172,6 +172,20 @@ def get_dataset(self, name):
             raise ValueError(
                 'Container \'{n}\` does not exist'.format(n=name))
 
+    def get_dataset_names(self):
+        """ Returns the names of all the CUDS objects in the containers.
+
+        """
+
+        ip = self._iter_particles()
```
Comment (itziakos) [inline simphony/io/h5_cuds.py]: how about using itertools.chain (https://docs.python.org/2/library/itertools.html#itertools.chain)

**Comment by itziakos** [comment]
two minor comments apart from that it looks ok :+1:

**Comment by roigcarlo** [comment]
@itziakos Changes made, its ok to merge?

**Comment by itziakos** [comment]
go ahead

**Pull request merged**